Makefile

```diff
@@ -61,6 +62,7 @@ pii_check: ## check for PII annotations on all Django models
 	tox -e pii_check
 
 requirements: ## install development environment requirements
+	pip install requirements/pip.txt
```

Comment:
This should be `pip install -r`, otherwise it's looking for a package named `requirements/pip.txt`.  And it looks like we're using `-qr` in most of the other repos we're updating.